### PR TITLE
DL-12295 Claim VAT Enrolment - Interstitial page

### PR DESCRIPTION
**Pull request by arunankannan**

### CODE REVIEW

**Comment by arunankannan** [comment]
Interstitial page - normal view

image

Interstitial page - welsh view

image

File: conf/application.conf [hunk 1]
```diff
@@ -335,6 +335,7 @@ urls {
     paye = "/paye-online"
     VATRegistration = "/vat-registration/how-to-register#when-you-cannot-register-online"
     onlineServicesHelpdesk = "/government/organisations/hm-revenue-customs/contact/online-services-helpdesk"
+    checkVATNumber = "/check-uk-vat-number"
```
Comment (chaithupejjai) [inline conf/application.conf:338]: May I know why is this required?

File: app/views/vat/claimVATInterstitialPage.scala.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    <h2 class="govuk-heading-s">@messages("claimVATInterstitialPage.p5.title")</h2>
+        <p class="govuk-body">@messages("claimVATInterstitialPage.p5.text1")</p>
+
+
+    <a data-module="govuk-button" id="continue"
+       href='@DoYouHaveVATRegNumberController.onPageLoad()'
+       class="govuk-button"
+       role="button">
+
```
Comment (chaithupejjai) [inline app/views/vat/claimVATInterstitialPage.scala.html:69]: Please format the code.

File: conf/application.conf (hunk 1 above)
Comment (arunankannan) [reply conf/application.conf:338]: please note, all gov links in views are added in the app config.  In this case, we have one for this text on the interstitial page.
"Check a UK VAT Number service (opens in new tab)"
Easy to change in future.

File: app/views/vat/claimVATInterstitialPage.scala.html (hunk 2 above)
Comment (arunankannan) [reply app/views/vat/claimVATInterstitialPage.scala.html:69]: sure, thanks

File: app/views/vat/claimVATInterstitialPage.scala.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```
Comment (chaithupejjai) [inline app/views/vat/claimVATInterstitialPage.scala.html:6]: Format and attachscreenshots in Jira ticket before change (staging) and after change (local) . ALl looks good to me.

Comment (arunankannan) [reply app/views/vat/claimVATInterstitialPage.scala.html:6]: sure, done now, thanks

**Review by chaithupejjai — APPROVED** [review]
